sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClient.java

```diff
@@ -20,22 +19,11 @@
  * specific Event Hub and allows operations for sending event data, receiving data, and inspecting the Event Hub's
  * metadata.
  *
- * <p>
- * Instantiated through {@link EventHubClientBuilder}.
- * </p>
- *
- * <p>
- * <strong>Creating a synchronous {@link EventHubClient} using an Event Hub instance connection string</strong>
- * </p>
- *
- * {@codesnippet com.azure.messaging.eventhubs.eventhubclient.instantiation}
- *
  * @see EventHubClientBuilder
  * @see EventHubAsyncClient To communicate with Event Hub using an asynchronous client.
  * @see <a href="https://docs.microsoft.com/Azure/event-hubs/event-hubs-about">About Azure Event Hubs</a>
  */
-@ServiceClient(builder = EventHubClientBuilder.class)
-public class EventHubClient implements Closeable {
+class EventHubClient implements Closeable {
```

Comment:
Maybe we can remove the package-private sync client now. Since EventProcessor uses only async client.

Reply:
It's used to create a synchronous receiver and consumer, so I kept it. :)